main.tf

```diff
@@ -17,7 +17,7 @@ data "google_client_config" "this" {}
 
 resource "google_spanner_instance" "default" {
   config           = var.config
-  display_name     = var.display_name
+  display_name     = var.name
```

Comment:
We can't remove this due to backward compatibility for Spanner instances like in RnD projects.  Can I suggest we default `display_name` to `name` if it's not set?

Reply:
`display_name` is fine to change, but legacy spanner instances have different `name` values.  Resolving this convo.